txscript/script.go

```diff
@@ -368,9 +539,9 @@ func calcSignatureHash(script []parsedOpcode, hashType SigHashType, tx *wire.Msg
 	// The final hash is the double sha256 of both the serialized modified
 	// transaction and the hash type (encoded as a 4-byte little-endian
 	// value) appended.
-	wbuf := bytes.NewBuffer(make([]byte, 0, txCopy.SerializeSize()+4))
```

Comment:
Oops, must've accidentally made this error during the initial set of commits. Reconciled in a fixup.